src/odemis/acq/fastem_conf.py

```diff
@@ -91,6 +91,15 @@ def configure_scanner(scanner, mode):
     # Immersion needs to be set before changing the horizontalFoV, as the range is updated
     scanner.immersion.value = conf["immersion"]
 
+    if scanner.immersion.value is False:
+        # Correct for shift in image between immersion mode and field free mode.
+        # If MD_FIELD_FREE_SHIFT is not provided fall back to a correction of [0, 0]
+        pos_cor = scanner.getMetadata().get(model.MD_FIELD_FREE_SHIFT, [0, 0])
```

Comment:
A good name for this metadata is a bit tricky to find. It'd be nice that it's clearly related to `POS_COR` and also to "immersion". `FIELD_FREE_POS` is odd (as it'd sound like an absolute value). `FIELD_FREE_POS_COR` is odd too. But I feel that `FIELD_FREE_SHIFT` doesn't clearly indicate it's a shift in *position* . How about `FIELD_FREE_POS_SHIFT`?